ipython/fragment_reattachment_example.ipynb

```diff
@@ -0,0 +1,1102 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d3349900",
+   "metadata": {},
+   "source": [
+    "# Random Reattachment of Fragment Species from Automatic Fragment Modeling (AFM) Simulation Results\n",
+    "AFM allows users to construct mechanisms for large molecules with RMG by breaking them up into two or more fragments. Running simulations with a fragment mechanism is much the same as any other mechanism. However, the species profiles obtained are for individual fragments that may be attached in many combinations. In this notebook, we give an example of random reattachment of fragments produced from polymer pyrolysis, which allows the user to obtain important observables such as molecular weight distribution or the approximate concentration of specific species composed of multiple fragments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "285ac939-646d-46e6-a5b9-286661e1ad18",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:No normalization for BCUT2D_MWHI\n",
+      "WARNING:root:No normalization for BCUT2D_MWLOW\n",
+      "WARNING:root:No normalization for BCUT2D_CHGHI\n",
+      "WARNING:root:No normalization for BCUT2D_CHGLO\n",
+      "WARNING:root:No normalization for BCUT2D_LOGPHI\n",
+      "WARNING:root:No normalization for BCUT2D_LOGPLOW\n",
+      "WARNING:root:No normalization for BCUT2D_MRHI\n",
+      "WARNING:root:No normalization for BCUT2D_MRLOW\n",
+      "WARNING:root:No normalization for AvgIpc\n"
+     ]
+    }
+   ],
+   "source": [
+    "from rdkit.Chem.rdmolops import *\n",
```

Comment:
Latest python style is to explicitly import the functions you are using, like `from rdkit.Chem.rdmolops import function_name` rather than `import *` - it just makes it more obvious where the various functions are coming from.